### ISSUE TITLE
Dragging the Timeline Progression bar creates a ghost image

### ISSUE THREAD

**Issue by DonovanPhoenix** [webtorrent/webtorrent-desktop#1892]
WebTorrent Desktop Version 0.24.0 (0.108.6) (x64)
Microsoft Windows [Version 10.0.19042.630]

If I try dragging the Timeline Progression bar it creates a ghost image
I added a picture to explain it better.

![Untitled](https://user-images.githubusercontent.com/40139772/98905855-ceffe980-24c4-11eb-91e7-89a41145282c.png)

**Comment by feross** [comment]
Confirmed. This didn't used to be an issue. Did this get introduced recently?

**Comment by dsernst** [comment]
Yes. I remember noticing this when I was working on the scrubbing preview a few months ago: https://github.com/webtorrent/webtorrent-desktop/pull/1812. But IIRC it was already happening before I started that.
